karma-demo.conf.js

```diff
@@ -18,9 +18,10 @@ module.exports = function (config) {
       require('karma-coverage-istanbul-reporter'),
       require('@angular-devkit/build-angular/plugins/karma')
     ],
-    // files: [
+     files: [
     //   {pattern: './scripts/test.ts', watched: false}
-    // ],
+         "node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
Why do you need this change?

Reply:
Short version: Without css available nothing would work.
Longer version: Animations are already defined by the css code. I try to not "reinvent the wheel" by not reimplementing bootstraps code in angular a second time. Actually I don't understand why it's often done in the project code base. Dosen't make sense to me.